doc/Repeater Architecture.md

```diff
@@ -4,7 +4,7 @@
 # Quantum Repeater Architecture #
 
 A repeater has exactly two interfaces, and is much like a
-[QNIC](QNIC.md) augmented with a [C4](C4.md).
+[QNIC](QNIC.md) augmented with a [C4](Classical Chassis Control Card (C4).md).
```

Comment:
Need to replace the ' 's with '%20'

Reply:
I reflexed your feedback, so could you check it? Thanks in advance.